src/AppInstallerCommonCore/Public/winget/ManifestCommon.h

```diff
@@ -130,12 +130,15 @@ bool HasExtension(std::string_view extension) const;
         AlreadyInstalled,
         Downgrade,
         BlockedByPolicy,
+        Custom
     };
 
     struct ExpectedReturnCode
     {
-        DWORD InstallerReturnCode;
-        ExpectedReturnCodeEnum ReturnResponse;
+        DWORD InstallerReturnCode{};
+        ExpectedReturnCodeEnum ReturnResponse{};
+        string_t ReturnResponseUrl{};
+        ExpectedReturnCode() {};

```

Comment:
>ExpectedReturnCode() {}; [](http://example.com/codeflow?start=8&length=24)

nit: above {} and constructor are probably over verbose for simple data type structs

Reply:
Removed constructor and {} from ReturnResponseUrl as it already has a default initializer.